README.md

```diff
@@ -6,11 +6,16 @@
 * [Composer](https://getcomposer.org)
 * [NodeJS](https://nodejs.org)
 * [Yeoman](http://yeoman.io)
+* [Gulp](http://gulpjs.com/)
 
 ```
 npm install -g yo
 ```
```

Comment:
@gvorbeck these can be combined into a single line:

`npm install -g yo gulp`

can you change that please?